draft-cwbgp-green-energy-saving-management.md

```diff
@@ -117,7 +117,7 @@ The document leverages types defined in {{?RFC3418}} and {{?RFC6933}}.
    modules, as shown in {{pref}}.
 
 | Prefix | YANG Module |  Reference   |
-| ianahw | iana-hardware          | [IANA_YANG] |
+| ianahw | iana-hardware          | {{!RFC8348}} |
```

Comment:
The OLD is correct as RFC8346 is only about the initial IANA module; the authoritative per rfc8407bis is the IANA URL.

Please see https://datatracker.ietf.org/doc/html/[draft-ietf-netmod-rfc8407bis-25](https://datatracker.ietf.org/doc/html/draft-ietf-netmod-rfc8407bis-25#name-iana-maintained-modules)#name-iana-maintained-modules for more details